Tasks/HelmInstallerV0/task.json

```diff
@@ -14,7 +14,7 @@
     "version": {
         "Major": 0,
         "Minor": 153,
```

Comment:
please change M153 to M154 and Patch to 1 as we have entered 154 sprint

Reply:
Got, it, done.